languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -24187,6 +24202,28 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <suggestion><match no="1" regexp_match="^[Rr]an" regexp_replace="heran"/></suggestion>
                 <example correction="heranzutasten">Er versuchte sich an das Problem <marker>ranzutasten</marker>.</example>
             </rule>
+            <rule>
```

Comment:
Das ist eine neue Subrule, oder? Dann bitte `<rule default="temp_off">`

Reply:
@udomai  temp_off drin
und habe es ans Ende der rulegroup gepackt.
wäre ready to merge